skimage/measure/_pnpoly.pyx

```diff
@@ -47,9 +47,10 @@ def _grid_points_in_poly(shape, verts):
     cdef cnp.ndarray[dtype=cnp.uint8_t, ndim=2, mode="c"] out = \
          np.zeros((M, N), dtype=np.uint8)
 
-    for m in range(M):
-        for n in range(N):
-            out[m, n] = point_in_polygon(V, &vx[0], &vy[0], m, n)
+    with nogil:
```

Comment:
indent  the nogil block 4 spaces for consistency?
Some of the edited files have 4 and some have 2

Reply:
Gah sorry, my editor went all wacky and I had to reset the configuration.... I didn't notice cython was set to 2 spaces until it was pretty late.

Reply:
done